src/gsCore/gsForwardDeclarations.h

```diff
@@ -63,199 +63,214 @@ enum gsNeedEnum
-template <class T=real_t>    class gsMVInterpolation;
-template <class T=real_t>    class gsVolumeSegment;
-template <class T=real_t>    class gsCompositeTopology;
-template <class T=real_t>    class gsBasisEvaluator;
-template <class T=real_t>    class gsMultiBasis;
+template <class T=real_t>                class gsOptParameterization;
+template <class T=real_t>                class gsQualityMeasure;
+template <class T=real_t>                class gsCuttingLoop;
+template <class T=real_t>                class gsInterpOption;
+template <class T=real_t>                class gsMVInterpolation;
+template <class T=real_t>                class gsVolumeSegment;
+template <class T=real_t>                class gsCompositeTopology;
+template <class T=real_t>                class gsBasisEvaluator;
+template <class T=real_t>                class gsMultiBasis;
 
-template <class T=real_t>    class gsBemLaplace;
-template <class T=real_t>    class gsBemSolution;
+template <class T=real_t>                class gsBemLaplace;
+template <class T=real_t>                class gsBemSolution;
 
-template <class T=real_t>    class gsBoundaryConditions;
+template <class T=real_t>                class gsBoundaryConditions;
 
 
-template <class T=real_t >  class gsVertex;
-template <class T=real_t >  class gsCell;
-template <class T=real_t >  class gsFace;
-template <class T=real_t >  class gsEdge;
+template <class T=real_t>                class gsVertex;
+template <class T=real_t>                class gsCell;
+template <class T=real_t>                class gsFace;
+template <class T=real_t>                class gsEdge;
 
-template <class T=real_t >  class gsHeVertex;
-template <class T=real_t >  class gsHalfFace;
-template <class T=real_t >  class gsHalfEdge;
-template <class T=real_t >  class gsMeshElement;
-template <class T=real_t >  class gsCurvatureSmoothing;
-template <class T=real_t >  class gsFitting;
+template <class T=real_t>                class gsHeVertex;
+template <class T=real_t>                class gsHalfFace;
+template <class T=real_t>                class gsHalfEdge;
+template <class T=real_t>                class gsMeshElement;
+template <class T=real_t>                class gsCurvatureSmoothing;
+template <class T=real_t>                class gsFitting;
 
 
-template< class T = real_t>  class gsCurveFitting;
+template <class T=real_t>                class gsCurveFitting;
 
-template<class T = real_t> struct gsNurbsCreator;
+template <class T=real_t>               struct gsNurbsCreator;
 
-template<class T = real_t> struct gsFieldCreator;
+template <class T=real_t>               struct gsFieldCreator;
 
 class gsOptionList;
 
-template<class T = real_t, int _Rows=-1, int _Cols=-1, 
-         int _Options  = 0|((_Rows==1 && _Cols!=1)?0x1:0)> class gsMatrix;
-template<class T = real_t, int _Rows=-1, int _Options = 0> class gsVector;
+template <class T=real_t, int _Rows=-1, int _Cols=-1,
+         int _Options  = 0|((_Rows==1 && _Cols!=1)?0x1:0)>
```

Comment:
wow

Reply:
that's not new, i just made some white space changes...